tests/usecases/data-processing-cloudant/manifest.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: data-processing-cloudant
+  version: 1.0
+  license: Apache-2.0
+  binding-package: /whisk.system/cloudant
+  inputs:
+    username: cloudant_user
+    password: cloudant_password
+    host: cloudant_user.cloudant.com
+  actions:
+    write-to-action:
+      version: 1.0
+      location: actions/write-to-cloudant.js
+      runtime: nodejs
+      inputs:
+        CLOUDANT_USERNAME: cloudant_user # The same as above one
+        CLOUDANT_PASSWORD: cloudant_password # The same as above one
+        CLOUDANT_DATABASE: cloudant_database
+    write-from-action:
+      version: 1.0
+      location: actions/write-from-cloudant.js
+      runtime: nodejs
+  sequences:
+      write-from-cloudant-sequence:
+        actions:  # cloudant_instance is the same as the package binding name
+  triggers:
+    image-uploaded:
+      feed: changes # cloudant_instance is the same as the package binding name
```

Comment:
The key feed doesn't do anything right now?  I implemented the feed action API which uses the key `source` and takes the fully qualified name as the value, in this case it would be `/whisk.system/cloudant/changes`.

I like the key `feed` better than `source.`. Should I use this for feed actions?